code/modules/antagonists/superheroes/buzzon/buzzon_equipment.dm

```diff
@@ -0,0 +1,294 @@
+	slot_flags = ITEM_SLOT_BACK
+	block_chance = 10 //It's about agility, not about sheer force
+	///If this var is higher than world.time this means that the owner is still in the blocking stance and has a high chance to block the attack
+	var/parry_time = 0
+
+/obj/item/melee/beesword/buzzon/hit_reaction(mob/living/carbon/human/owner, atom/movable/hitby, attack_text = "the attack", final_block_chance = 0, damage = 0, attack_type = MELEE_ATTACK)
+	if(parry_time >= world.time)
+		parry_time = 0
+		final_block_chance += 85
+
+	switch(attack_type)
+		if(UNARMED_ATTACK)
+			final_block_chance += 30
+
+		if(PROJECTILE_ATTACK)
+			final_block_chance -= 40
+
+		if(LEAP_ATTACK)
+			if(isliving(hitby))
+				var/mob/living/victim = hitby
+				force = 20
+				attack_verb_continuous = list("penetrates", "pierces") //We pierce them in the flight.
+				attack_verb_simple = list("penetrate", "pierce")
+				attack(victim, owner)
+				force = initial(force)
+				attack_verb_continuous = initial(attack_verb_continuous)
+				attack_verb_simple = initial(attack_verb_simple)
+	. = ..()
+
+/obj/item/melee/beesword/buzzon/attack_secondary(mob/living/victim, mob/living/user, params)
+	user.changeNext_move(CLICK_CD_RANGE)
+	parry_time = world.time + CLICK_CD_RANGE
+	user.visible_message("<span class='warning'>[user] assumes blocking stance.</span>", "<span class='notice'>You assume blocking stance.</span>")
+	return SECONDARY_ATTACK_CANCEL_ATTACK_CHAIN
+
+/obj/item/melee/beesword/buzzon/throw_impact(atom/hit_atom, datum/thrownthing/throwingdatum)
+	if(ishuman(hit_atom))
+		var/mob/living/carbon/human/victim = hit_atom
+		if(istype(victim.get_item_by_slot(ITEM_SLOT_OCLOTHING), /obj/item/clothing/suit/hooded/bee_costume/buzzon))
+			var/obj/item/clothing/suit/hooded/bee_costume/buzzon/suit = victim.get_item_by_slot(ITEM_SLOT_OCLOTHING)
+			if(suit.linked_sword == src)
+				victim.put_in_active_hand(src)
+				victim.visible_message("<span class='warning'>[victim] catches [src] out of the air!</span>")
+				return
+	. = ..()
+
+/mob/living/simple_animal/hostile/bee/toxin_type
+	var/toxin_type = /datum/reagent/toxin
+
+/mob/living/simple_animal/hostile/bee/toxin_type/Initialize()
+	. = ..()
+	assign_reagent(GLOB.chemical_reagents_list[toxin_type])
+
+/mob/living/simple_animal/hostile/bee/toxin_type/cryo
+	toxin_type = /datum/reagent/consumable/frostoil
+
+/obj/item/grenade/spawnergrenade/buzzkill/non_toxic
+	spawner_type = /mob/living/simple_animal/hostile/bee
+	deliveryamt = 5
+
+/obj/item/clothing/glasses/hud/security/sunglasses/blue
+	icon_state = "sunhudmed"
+	glass_colour_type = /datum/client_colour/glass_colour/blue
+
+/obj/item/melee/beesword/buzzon/cryo
+	name = "The Cryostinger"
+	desc = "An advanced sword with built-in frost oil injectors."
+	icon_state = "buzzon_sword_cryo"
+	inhand_icon_state = "buzzon_cryo"
+	worn_icon_state = "buzzon_cryo"
+	force = 3 //A bit lower because we have better toxin
+
+	toxin_type = /datum/reagent/consumable/frostoil
```

Comment:
ironically, despite being a roboticist, cold!buzzon is absolutely terrible at fighting full-augged people (as their damage reduction no-sells the direct damage from his sword, and their effective immunity to cold damage makes his frost oil deal no damage either), and true androids can dunk on all three variants of buzzon (as they don't process reagents). I think their bees might or might not be able to attack and inject chems into full-augged people (because beecode is an atrocity), but even if they can, a full-augged person can just wear some thick clothing to become immune to bees.

I'd recommend giving buzzon an emp implant or something (maybe some kind of self-recharging one?) so that he can hardstun and then cuff full-augged people (and/or wail on some emp'd cyborgs for a bit).

Reply:
Yeah, that's true...

Reply:
I don't think that EMP implants are what he needs, will think some more about this problem